CHANGELOG-1.x.md

```diff
@@ -3,6 +3,7 @@
 
 ## vN.N.N
 * Use `Swal.DismissReason` enum in `receiveEnvelope.js` and `Authorizer.js`
+* Fix Swipes disappearing after save
```

Comment:
@hunyoboy can you make this a bit more descriptive/accurate? what we're really talking about here is the form fields are not hydrating properly, right?

Reply:
@efjacobson updated